pyro/infer/autoguide/effect.py

```diff
@@ -213,20 +213,20 @@ def _get_posterior_median(self, name, prior):
 
 class AutoHierarchicalNormalMessenger(AutoNormalMessenger):
     """
-    EXPERIMENTAL Automatic :class:`~pyro.infer.effect_elbo.GuideMessenger` ,
-    intended for use with :class:`~pyro.infer.effect_elbo.Effect_ELBO` or
-    similar.
+    :class:`AutoMessenger` with mean-field normal posterior conditional on all dependencies.
 
     The mean-field posterior at any site is a transformed normal distribution,
     the mean of which depends on the value of that site given its dependencies in the model:
```

Comment:
double colon before a code block:
```
the mean of ... model::

    loc_total = ...
```

Reply:
Sorry, still learning how to use pandoc syntax